lib/media/media_source_engine.js

```diff
@@ -212,22 +212,24 @@ shaka.media.MediaSourceEngine.prototype.destroy = function() {
  *   MIME types.  For example: { 'audio': 'audio/webm; codecs="vorbis"',
  *   'video': 'video/webm; codecs="vp9"', 'text': 'text/vtt' }.
  *   All types given must be supported.
+ * @param {shakaExtern.StreamingConfiguration} config
  *
  * @throws InvalidAccessError if blank MIME types are given
  * @throws NotSupportedError if unsupported MIME types are given
  * @throws QuotaExceededError if the browser can't support that many buffers
  *
  * @suppress {unnecessaryCasts}
  */
-shaka.media.MediaSourceEngine.prototype.init = function(typeConfig) {
+shaka.media.MediaSourceEngine.prototype.init = function(typeConfig, config) {
```

Comment:
Let's just pass a useRelativeCueTimestamps boolean down instead of the whole config structure. This means we won't be able to update it, but given the nature of the setting this should be fine.